incuna-sass/components/_datepicker.sass

```diff
@@ -1,5 +1,5 @@
 .datepicker
-    $datepicker-button-size: rem(32px)
+    $datepicker-button-size: rem(32px) !default
```

Comment:
makes it easier to overwrite I think, I mostly added it for consistency, we have it on most of the other variables in the library, so we should have it on all really :)